### PR TITLE
yatra assignment in robot framework

### PR DESCRIPTION
**Pull request by x-akarsh-x**
…e review it in your free time.
Thank you.

### CODE REVIEW

**Comment by x-akarsh-x** [comment]
Hello Guys
This is my assignment of yatra in robot framework.
Please review it in your free time.
Thank you.

File: Akarsh/yatra/Configration/env_yatra.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+*** Settings ***
+Documentation       These Are The Variables That Are Used In rosource_yatra
+
+*** Variables ***
+
```
Comment (pcube0608) [inline Akarsh/yatra/Configration/env_yatra.txt]: remove unwanted new lines

File: Akarsh/yatra/Tests/tc_yatra.txt
Comment (pcube0608) [inline Akarsh/yatra/Tests/tc_yatra.txt:1]: The extension of file should be .robot

File: Akarsh/yatra/Resources/resource_yatra.txt
Comment (pcube0608) [inline Akarsh/yatra/Resources/resource_yatra.txt:1]: The extension of file should be .robot

File: Akarsh/yatra/Configration/env_yatra.txt
Comment (pcube0608) [inline Akarsh/yatra/Configration/env_yatra.txt:1]: The extension of file should be .robot

**Review by pcube0608 — CHANGES_REQUESTED** [review]
There should not be any folder called yatra in Akarsh.
The folder structure should be 
```
Akarsh
 - Configuration
 - Resources
 - Tests
              -   TC_Yatra.robot
              -   TC_SauceDemo.robot
```

**Comment by x-akarsh-x** [comment]
> There should not be any folder called yatra in Akarsh. The folder structure should be
> 
> ```
> Akarsh
>  - Configuration
>  - Resources
>  - Tests
>               -   TC_Yatra.robot
>               -   TC_SauceDemo.robot
> ```

**Pull request closed without merging**